Content.Server/Spawners/Components/TimedSpawnerComponent.cs

```diff
@@ -9,24 +8,15 @@ namespace Content.Server.Spawners.Components
     [RegisterComponent]
     public sealed partial class TimedSpawnerComponent : Component, ISerializationHooks
     {
-        [ViewVariables(VVAccess.ReadWrite)]
-        [DataField("prototypes", customTypeSerializer:typeof(PrototypeIdListSerializer<EntityPrototype>))]
+        [DataField(customTypeSerializer: typeof(PrototypeIdListSerializer<EntityPrototype>))]
         public List<string> Prototypes { get; set; } = new();
 
-        [ViewVariables(VVAccess.ReadWrite)]
-        [DataField("chance")]
```

Comment:
this is a regression keep [DataField] on all of these, and while you are here remove `{ get; set; }` for them all

Reply:
Oh yeah, whoops. I meant to just delete the field name, not the whole attribute.